test/minicluster.py

```diff
@@ -17,13 +17,20 @@
 
 import getpass
 import os
-import unittest
 
-from snakebite.minicluster import MiniCluster
-
-from luigi import hadoop, hdfs
+from luigi import hadoop, hdfs, six
 from nose.plugins.attrib import attr
 
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+if six.PY3:
+    raise unittest.SkipTest("snakebite doesn't work on Python yet.")
```

Comment:
By doing this we also skip the definition of `class MiniClusterHadoopJobRunner` which is needed by hadoop_test